examples/step-69/step-69.cc

```diff
@@ -2863,9 +2867,16 @@ namespace Step69
     // At this point we can return from the <code>output()</code> function
     // and resume with the time stepping in the main loop - the thread will
     // run in the background.
-    if (!asynchronous_writeback)
+    if (asynchronous_writeback)
       {
+#ifdef DEAL_II_WITH_THREADS
         background_thread_state = std::async(std::launch::async, output_worker);
+#else
+        AssertThrow(
+          false,
+          ExcMessage(
+            "\"asynchronous_writeback\" was set to true but deal.II was built without thread support (\"DEAL_II_WITH_THREADS=false\")."));
```

Comment:
Doesn't something like this work?
```suggestion
            "\"asynchronous_writeback\" was set to true but deal.II was built "
            "without thread support (\"DEAL_II_WITH_THREADS=false\")."));
```
We do it in many other places.

Reply:
This works!